packages/runtime/src/controls/control.ts

```diff
@@ -28,6 +28,7 @@ export type ControlDefinition =
   | ShapeControlDefinition
   | ListControlDefinition<any>
   | LinkControlDefinition
+  | SlotControlDefinition
```

Comment:
This will allow `Slot` as a valid type for `Shape`, I think, which we might not want just yet.

Reply:
For this case we can make a new type, just for `Shape` that omits the `Slot` definition or something like that. It's also not a big deal, `Slot` in `Shape` should be released together with this anyway.

I don't like that we're avoiding writing change logs because it's tied to a release. Those two things should be decoupled from one another... I'm seriously considering removing the release pull request automation in favor of manual releases so that we can write change logs without worrying about it making its way to a release.

I say just write the change log for what you're doing and if you don't intend it to be used or don't want to release it then don't expose it in the public API or expose it under an `unstable_` prefix.

Reply:
Things could also be placed behind an experimental or unstable feature flag. Do whatever you think is best.